app/webpack.config.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     // Create aliases to import or require certain modules more easily
     alias: {
       // these ensure we don't have duplicate versions on the same page
+      '@bayer/ol-kit': path.resolve(__dirname, '../', 'src'),
```

Comment:
this allows us to use the same paths as external consumers within the demo/test app